src/main/scala/org/hammerlab/guacamole/assembly/AssemblyUtils.scala

```diff
@@ -0,0 +1,168 @@
+package org.hammerlab.guacamole.assembly
+
+import htsjdk.samtools.CigarOperator
+import org.apache.spark.Logging
+import org.hammerlab.guacamole.alignment.{AffineGapPenaltyAlignment, ReadAlignment}
+import org.hammerlab.guacamole.reads.MappedRead
+import org.hammerlab.guacamole.reference.{ContigSequence, ReferenceGenome}
+import org.hammerlab.guacamole.util.CigarUtils
+import org.hammerlab.guacamole.variants.ReferenceVariant
+import org.hammerlab.guacamole.windowing.SlidingWindow
+
+import scala.collection.JavaConversions._
+
+
+object AssemblyUtils extends Logging {
```

Comment:
I would name this `….assembly.Utils` instead of `….assembly.AssemblyUtils`, per discussion on #463, but if you remain unconvinced then this wfm 😄